systemd/jenkins.service

```diff
@@ -0,0 +1,142 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+Requires=network.target
+After=network.target
+
+#
+# Note: Do not customize this section. Instead, place your customizations in
+# the following drop-in file:
+#
+#     /etc/systemd/system/@@ARTIFACTNAME@@.service.d/override.conf
```

Comment:
Perhaps it should mention `systemctl edit @@ARTIFACTNAME.service@@` as a way to customize.

Reply:
Fine, done in the latest commits